app/javascript/leaflet/viewers/oembed.js

```diff
@@ -0,0 +1,21 @@
+import LeafletViewerBase from "./base.js";
+
+export default class LeafletViewerOembed extends LeafletViewerBase {
+  constructor() {
+    super();
```

Comment:
Need to add constructor(element) + super(element). Currently not working. http://localhost:3000/catalog/stanford-dc482zx1528